portal-web/docroot/html/taglib/ui/search_iterator/lexicon/bottom.jspf

```diff
@@ -94,39 +94,112 @@
 
 		<c:if test="<%= fixedHeader && displayStyle.equals(SearchIteratorTag.DEFAULT_DISPLAY_STYLE) %>">
 			var contentBox = searchContainer.get('contentBox');
-
-			var fixedHeader = document.getElementById('<%= namespace + id %>fixedHeader');
+			var insideIframe = window !== Liferay.Util.getOpener();
 
 			if (contentBox) {
+				var tableElement;
+
+				var fixedHeader = document.getElementById('<%= namespace + id %>fixedHeader');
 				var mainContent = contentBox.ancestor('#main-content');
+				var tableResponsive = contentBox.one('.table-responsive');
 
-				if (mainContent) {
-					var scrollingContainer = window === Liferay.Util.getOpener() ? window : mainContent._node;
+				var registerEventListener = function(element, eventType, eventListener) {
+					element.addEventListener(eventType, eventListener);
+
+					Liferay.on(
+						'destroyPortlet',
+						function() {
+							element.removeEventListener(eventType, eventListener);
+						}
+					);
+				};
 
-					var table = fixedHeader.parentElement.parentElement;
+				if (mainContent) {
+					var scrollingContainer = insideIframe ? mainContent._node : window;
```

Comment:
We should be able to use `mainContent.getDOM()` instead of `mainContent._node`.  We should not access private properties and methods if possible.